### ISSUE TITLE
Bar Chart: VPAT Accessibility issues

### ISSUE THREAD

**Issue by JBismonte** [infor-design/enterprise#6074]
**Describe the bug**
Bar charts do not provide textual alternatives.
Information in bar charts including x and y axes and bars is contained in an svg without text alternative. Text fill values are announced in some browser/screenreader combinations, but cannot be relied upon as a textual alternative. The actual values for each bar, displayed in a tooltip on mouse hover (e.g. "Category B 372", "Category C 236.35") is not announced to screen reader users. 

**To Reproduce**

1. Go to https://main-enterprise.demo.design.infor.com/components/bar/example-index.html
2.  bar chart cannot be focused by the keyboard and when using a screen reader, its tooltips are not announced when the mouse is hovered in each of the bar.

**Recommendation by VPAT**
Developers must ensure that information conveyed in charts and graphs is provided textually.
Consider also providing the information in a table alongside the chart. This approach allows all users to see the information presented in a more structured way.
If this is not possible, the table alternative could be visually hidden, and only presented to screen reader users.
If it is necessary to instead try to make the SVG fully accessible, refer to the following resources that may be of use:
* https://css-tricks.com/accessible-svgs/
* http://simplyaccessible.com/article/7-solutions-svgs/

**Version**

- ids-enterprise: 4.60

**Screenshots**
![image](https://user-images.githubusercontent.com/29673378/151765605-15829018-e3ff-4612-a7b2-01f68252d506.png)

**Platform**
 - Infor Application/Team Name: Landmark Client
 - Device: (if applicable) : NA
 - OS Version: Windows 10
 - Browser Name: Firefox
 - Browser Version: [e.g. 22, 66.0.3359.181 (Official Build) (64-bit)]

**Additional context**
https://jira.lawson.com/browse/LMCLIENT-34217

**Comment by tmcconechy** [comment]
Plan for this:

- solve as a list in WC version
- provide example here with the same (might need help from @ericangeles )
- will implement for just bar code so that it can be verified before adding to any other components here

**Comment by tmcconechy** [comment]
If you see what i did on https://main.wc.design.infor.com/ids-bar-chart/example.html with aria i think we should implement the same in the EP bar chart.

- a title in the svg
- a parent element with role="list" and an aria-label
- group each bar in a <g> with role="lisitem"inside that have the bar but also an offscreen text element with all the info in it
- Then when you try it with a screen reader you navigate to the chart with the virtual cursor then can read through the chart data like a list

I think we should do it just for bar chart for now so they can confirm the approach with level access

**Comment by janahintal** [comment]
This issue is now resolved in https://main-enterprise.demo.design.infor.com/components/bar/example-index.html. Thanks.
